resources/profiles/Qidi/filament/Qidi Generic ASA.json

```diff
@@ -0,0 +1,21 @@
+{
+    "type": "filament",
+    "filament_id": "GFB99",
+    "setting_id": "GFSA04",
+    "name": "Qidi Generic ASA",
+    "from": "system",
+    "instantiation": "true",
+    "inherits": "fdm_filament_asa",
+	"default_filament_colour": "#283688",

```

Comment:
Can you delete this line from all filament profiles

Reply:
You mean the line for the default colour of the filament "default_filament_colour": "#283688", ?

Reply:
Yes